tests/unit/compiler/nginx/init_test.py

```diff
@@ -49,25 +46,22 @@ def test_get_nginx_configuration_spec_2(self):
                     proxy_set_header Connection "upgrade";
                     proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
                     proxy_set_header Host $host;
-                    proxy_pass http://127.0.0.0:8000;
+                    proxy_pass http://172.17.42.1:8000;
```

Comment:
Generally try to avoid using variables in the expected test output, this way you're actually writing down what you expect to see.